ember-electron/test-main.js

```diff
@@ -14,23 +16,30 @@ let [, , indexUrl] = process.argv;
 indexUrl = indexUrl.replace(/__amp__/g, '&');
 console.log(`URL: ${indexUrl}`);
 let {
-  pathname: indexPath,
+  pathname: originalIndexPath,
   search: indexQuery,
 } = url.parse(indexUrl);
 // When we extract the pathname from an absolute path on windows, it starts
 // with '/C:/', and the leading slash confuses everything, so we need to strip
 // it.
 if (process.platform === 'win32') {
-  indexPath = indexPath.slice(1);
+  originalIndexPath = originalIndexPath.slice(1);
 }
-indexPath = resolve(indexPath);
+
+// Copy index-electron.html to a location not managed by broccoli so that it
+// doesn't get deleted when a change is detected.
+originalIndexPath = resolve(originalIndexPath);
+let emberAppDir = resolve(dirname(originalIndexPath), '..');
+let tmpDir = tmp.dirSync({ dir: resolve(emberAppDir, '../../'), unsafeCleanup: true }).name;
```

Comment:
Oh sorry, from your comment I thought you wanted me to keep the `dir` option. I'll remove that and `unsafeCleanup` - I wasn't sure if that was needed since we're creating a file in the tmp dir. Good to know.